EntryRenderer crash when MainWindow get focus

<!-- Bug report best practices: https://github.com/xamarin/Xamarin.Forms/wiki/Submitting-Issues -->

### Description
EntryRenderer crash when MainWindow get focus.

### Steps to Reproduce
1.  Add an Entry to Page in XAML.
2. Add this page to NavigationPage.
3. Push another page to NavigationPage.
4. Make window lost focus.
5. Make windows has focus.
6. Crash.



### Basic Information

- Version with issue: 
Xamarin.Forms 3.2.0.871581
- Last known good version: 
unknown
- IDE: 
Visual Studio for mac
- Platform Target Frameworks: <!-- All that apply -->
macOS
- Nuget Packages:
Xamarin.Forms
- Affected Devices:
macOS Mojave

### call stack
Unhandled Exception:
System.NullReferenceException: Object reference not set to an instance of an object
  at Xamarin.Forms.Platform.MacOS.EntryRenderer+FormsNSTextField.HandleWindowDidBecomeKey (System.Object sender, System.EventArgs args) [0x0000c] in <6cd9e28826dd4c9ab5dd6113c0202c8f>:0 
  at AppKit.NSWindow+_NSWindowDelegate.DidBecomeKey (Foundation.NSNotification notification) [0x00011] in /Library/Frameworks/Xamarin.Mac.framework/Versions/5.0.0.0/src/Xamarin.Mac/AppKit/NSWindow.g.cs:11486 
  at (wrapper managed-to-native) AppKit.NSApplication.NSApplicationMain(int,string[])
  at AppKit.NSApplication.Main (System.String[] args) [0x00040] in /Library/Frameworks/Xamarin.Mac.framework/Versions/5.0.0.0/src/Xamarin.Mac/AppKit/NSApplication.cs:100 
  at mac.MainClass.Main (System.String[] args) [0x00007] in /Users/someone/Projects/TestMacCrash/mac/Main.cs:10 
[ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
  at Xamarin.Forms.Platform.MacOS.EntryRenderer+FormsNSTextField.HandleWindowDidBecomeKey (System.Object sender, System.EventArgs args) [0x0000c] in <6cd9e28826dd4c9ab5dd6113c0202c8f>:0 
  at AppKit.NSWindow+_NSWindowDelegate.DidBecomeKey (Foundation.NSNotification notification) [0x00011] in /Library/Frameworks/Xamarin.Mac.framework/Versions/5.0.0.0/src/Xamarin.Mac/AppKit/NSWindow.g.cs:11486 
  at (wrapper managed-to-native) AppKit.NSApplication.NSApplicationMain(int,string[])
  at AppKit.NSApplication.Main (System.String[] args) [0x00040] in /Library/Frameworks/Xamarin.Mac.framework/Versions/5.0.0.0/src/Xamarin.Mac/AppKit/NSApplication.cs:100 
  at mac.MainClass.Main (System.String[] args) [0x00007] in /Users/someone/Projects/TestMacCrash/mac/Main.cs:10

--- a/Xamarin.Forms.Platform.MacOS/Renderers/EntryRenderer.cs
+++ b/Xamarin.Forms.Platform.MacOS/Renderers/EntryRenderer.cs
@@ -77,7 +77,7 @@ namespace Xamarin.Forms.Platform.MacOS
 
 			void HandleWindowDidBecomeKey(object sender, EventArgs args)
 			{
-				if (CurrentEditor == Window.FirstResponder)
+				if (Window != null && CurrentEditor == Window.FirstResponder)
 					FocusChanged?.Invoke(this, new BoolEventArgs(true));
 			}
 		}

